plugin/src/test/java/com/spotify/plugin/dockerfile/TagsSelectorTest.java

```diff
@@ -0,0 +1,72 @@
+/*-
+ * -\-\-
+ * Dockerfile Maven Plugin
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.plugin.dockerfile;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.junit.Test;
+
+public class TagsSelectorTest {
+
+  @Test
+  public void validateTagsAreSelectedCorrectly() {
+
+    TagsSelector selector = new TagsSelector();
+    List<String> tags;
+
+    // Build
+    //
+    // <tags>
+    //   <tag>latest</tag>
+    // </tags>
+    //
+    tags = selector.select(ImmutableList.of("latest"), null, null);
+    assertThat(tags.size()).isEqualTo(1);
```

Comment:
nit: you could use https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/ListAssert.html#containsExactly-ELEMENT...-

Reply:
Yes, that's nicer.